packages/developer-portal/src/components/pages/app-detail/pipeline/deployment-info.tsx

```diff
@@ -0,0 +1,105 @@
+import { cx } from '@linaria/core'
+import {
+  BodyText,
+  Button,
+  ButtonGroup,
+  ColSplit,
+  elMb6,
+  InputWrap,
+  Intent,
+  StatusIndicator,
+  Subtitle,
+  Table,
+  TableCell,
+  TableHeader,
+  TableHeadersRow,
+  TableRow,
+  Title,
+  Grid,
+} from '@reapit/elements'
+import { PipelineModelInterface, PipelineRunnerModelInterface } from '@reapit/foundations-ts-definitions'
+import React from 'react'
+
+const buildStatusToIntent = (status: string): Intent => {
+  switch (status) {
+    case 'CREATING_ARCHITECTURE':
+      return 'primary'
+    case 'SUCCESS':
+      return 'primary'
+    case 'RUNNING':
+      return 'secondary'
+    case 'PENDING':
+    case 'QUEUED':
+    default:
+      return 'neutral'
+  }
+}
+
+const buildStatusToReadable = (status: string): string =>
+  status
+    .split('_')
+    .map((str) => str.toLowerCase())
+    .join(' ')
+
+const PipelineInfo = ({ pipeline }: { pipeline: PipelineModelInterface }) => (
```

Comment:
Can you define an interface for this please

Reply:
does it need one?